src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -2860,6 +2860,9 @@ A catch() block after a catch (System.Exception e) block can catch non-CLS excep
   <data name="ERR_BadArgExtraRef" xml:space="preserve">
     <value>Argument {0} may not be passed with the '{1}' keyword</value>
   </data>
+  <data name="ERR_BadArgExtraRefLangVersion" xml:space="preserve">
```

Comment:
>ERR_BadArgExtraRefLangVersion [](http://example.com/codeflow?start=14&length=29)

Since this is a new language version specific error, we need to make changes in CSharpUpgradeProjectCodeFixProvider.cs and add corresponding test in UpgradeProjectTests.cs